_playwright-tests/tests/API/CheckGetFeatures.spec.ts

```diff
@@ -0,0 +1,6 @@
+import { test, expect } from '@playwright/test';
+
+test('Content > getFeatures API', async ({ request }) => {
+    const result = await request.get('/api/content-sources/v1/repositories/');
```

Comment:
i know this is just a sample test, but this actually isn't testing the features api :)   should be /api/content-sources/v1//features/

Reply:
oh derp
I think I had the features api originally and then switched to this.. :|